distributed/tests/test_worker.py

```diff
@@ -1270,6 +1339,7 @@ def __reduce__(self):
 async def test_spill_no_target_threshold(c, s, a):
     """Test that you can enable the spill threshold while leaving the target threshold
     to False
+    If running locally pass --leaks=fds,processes,threads
```

Comment:
@crusaderky I noticed that without adding those flags the tests don't pass locally, I added a comment to mention that.

Reply:
fixed in main